optimade.md

```diff
@@ -1,942 +1,11 @@
-See the [github repository](https://github.com/Materials-Consortia/OPTiMaDe) for the most up-to-date specification.
+# OPTiMaDe API Specification
 
-# OPTIMADE API specification v0.9.5
+**The latest _stable_ version** is found in the [master](https://github.com/Materials-Consortia/OPTiMaDe/tree/master) branch in the GitHub repository.  
```

Comment:
one last comment: By linking to the repository and not the specification, you are introducing 2 extra clicks in total.
one might consider linking to the specification here directly.